### PR TITLE
Updated firecracker go SDK to last version

### PR DESCRIPTION
**Pull request by Zyqsempai**
Signed-off-by: bpopovschi <zyqsempai@mail.ru>
Issue #45

*Description of changes:*
Updated firecracker go SDK to last version

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -1,9 +1,9 @@
 module github.com/firecracker-microvm/firectl
 
 require (
-	github.com/firecracker-microvm/firecracker-go-sdk v0.17.0
+	github.com/firecracker-microvm/firecracker-go-sdk v0.17.1-0.20191030150325-dbf9a1e05f090
```
Comment (nmeyerhans) [inline go.mod]: We should do a proper SDK release so we can reference something like "v0.19.0" in this go.mod file...

Comment (nmeyerhans) [reply go.mod]: v0.19.0 of the SDK is released. Can you update this PR to use it?

Comment (Zyqsempai) [reply go.mod]: Done, but have strange test failings.

File: options_test.go [hunk 2]
```diff
@@ -549,14 +553,18 @@ func TestGetFirecrackerNetworkingConfig(t *testing.T) {
...
+					},
+					AllowMMDS: false,
 				},
 				firecracker.NetworkInterface{
-					MacAddress:  "morethings",
-					HostDevName: "morevalid",
-					AllowMMDS:   false,
+					StaticConfiguration: &firecracker.StaticNetworkConfiguration{
+						MacAddress:  "things",
+						HostDevName: "valid",
```
Comment (nmeyerhans) [inline options_test.go]: And this should be "morevalid"

**Comment by nmeyerhans** [comment]
So, the error that you're seeing the tests is what firectl shows when the configured firecracker executable doesn't exist. By default it looks for "firecracker" via PATH. I've opened #48 to update the test environment to ensure that "firecracker" does exist in the PATH. If you rebase against the commits in that PR, your tests should pass.

We should improve our testing around the handling of the firecracker binary location. Expansion of PATH should work (and we should provide a better error in case it doesn't!) and we should ensure that specifying alternate location via the `--firecracker-binary` option also works.

**Comment by Zyqsempai** [comment]
@nmeyerhans Great, thank you, PTAL

**Comment by Zyqsempai** [comment]
@nmeyerhans Ping!

**Comment by nmeyerhans** [comment]
Merged as f3cbc14d09eb214f95fa6a743767df2cb4ed739f. It got a different SHA because the buildkite changes you rebased over got a different SHA themselves by the time they were merged. I dropped that commit, and just merged your change.

**Pull request closed without merging**